### PR TITLE
ci: draft of it test for monorepo

### PR DESCRIPTION
**Pull request by fpaul-1A**
## Proposed change

Add an IT test with monorepo for o3r/analytics and @o3r/core

## Related issues

- :bug: Fixes #(issue)
- :rocket: Feature #(issue)

### CODE REVIEW

File: packages/@o3r/test-helpers/src/utilities/package-manager.ts [hunk 1]
```diff
@@ -37,13 +43,14 @@ export function getPackageManager() {
...
   try {
+    const d = Date.now();
     const output = execFileSync(
       cmd.split(' ')[0],
       [...cmd.split(' ').slice(1), ...args.split(' ')].filter((arg) => !!arg),
       {...options, shell: process.platform === 'win32', stdio: 'pipe', encoding: 'utf8'}
     );
     // eslint-disable-next-line no-console
-    console.log(`${cmd} ${args}\n${output}`);
+    console.log(`${cmd} ${args} [${Date.now() - d}ms]\n${output}`);
```
Comment (kpanot) [inline packages/@o3r/test-helpers/src/utilities/package-manager.ts:53]: Using performance API would be cleaner :)

File: packages/@o3r/test-helpers/src/utilities/package-manager.ts [hunk 2]
```diff
@@ -85,6 +92,19 @@ export function packageManagerExec(script: string, options: ExecSyncOptions) {
...
 }
 
+/**
+ * Execute a binary command (npx / yarn exec) for a specific workspace
+ * @param workspace
+ * @param script
+ * @param options
+ */
+export function packageManagerWorkspaceExec(workspace: string, script: string, options: ExecSyncOptions) {
+  if (getPackageManager() === 'npm') {
```
Comment (kpanot) [inline packages/@o3r/test-helpers/src/utilities/package-manager.ts:102]: Could be moved out as function (as it is used twice)

File: packages/@o3r/test-helpers/src/utilities/package-manager.ts [hunk 3]
```diff
@@ -15,14 +15,18 @@ const PACKAGE_MANAGERS_CMD = {
     create: 'npm create',
     exec: 'npm exec',
     install: 'npm install',
-    run: 'npm run'
+    run: 'npm run',
+    workspaceExec: 'npm exec -w ',
```
[the remark this answers is not in the document]
Comment (fpaul-1A) [reply packages/@o3r/test-helpers/src/utilities/package-manager.ts]: my mistake

File: packages/@o3r/test-helpers/src/utilities/package-manager.ts (hunk 1 above)
Comment (fpaul-1A) [reply packages/@o3r/test-helpers/src/utilities/package-manager.ts]: done

File: packages/@o3r/test-helpers/src/utilities/package-manager.ts (hunk 2 above)
Comment (fpaul-1A) [reply packages/@o3r/test-helpers/src/utilities/package-manager.ts:112]: done

**Review by kpanot — APPROVED** [review]

**Review by mrednic-1A — APPROVED** [review]

**Pull request merged**